### PR TITLE
Added additional information about managing gift registry items

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information about managing gift registry items from customer's account

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-gift-registry.html

whatsnew
Updated the [Gift Registry](https://docs.magento.com/user-guide/customers/account-dashboard-gift-registry.html) topic to add the workflow for customers managing their gift registry items from their customer dashboard.

### CODE REVIEW

File: src/customers/account-dashboard-gift-registry.md [hunk 1]
```diff
@@ -44,6 +44,19 @@ _Gift Registry_
...
 1. Edit the required options, and click **Save**.
 
+## Manage gift registry items
+
+![]({% link images/images-ee/account-dashboard-gift-registry-items-management.png %}){: .zoom}
+_Gift Registry items managing_
+
+1. From customer account dashboard, go to **Gift Registry**.
+
+1. Find the registry event, items which you want to manage, and click **Manage Items**.
```
Comment (BarnyShergold) [inline src/customers/account-dashboard-gift-registry.md:54]:
```suggestion
1. Find the registry event, then the items which you want to manage, and click **Manage Items**.
```

File: src/customers/account-dashboard-gift-registry.md [hunk 2]
```diff
@@ -44,6 +44,19 @@ _Gift Registry_
...
+
+1. Change the needed options like: **Note**, **Qty** or **Delete** an item from gift registry by checking checkbox near needed item.
```
Comment (BarnyShergold) [inline src/customers/account-dashboard-gift-registry.md:56]: Suggest breaking up the line as it read a bit confused as there are two different types of action on the line.
```suggestion
1. Change the required options such as : **Note** &  **Qty** 

1.  Or **Delete** an item from the gift registry by checking the checkbox near the required item.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/customers/account-dashboard-gift-registry.md (hunk 2 above)
Comment (DmitryTsymbal) [reply src/customers/account-dashboard-gift-registry.md]: Thank you for the review. Changes applied.

**Review by BarnyShergold — APPROVED** [review]
Thanks & approved!

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**